elastic/beats
# [Kubernetes] Deploy state namespace metricset

## What

Metricset `state_namespace` is introduced in 8.11 version.
This PR is just to update the metricbeat manifest files so `state_namespace` is there as default along with all the other kubernetes metricsets.


## How to test this PR locally

1. Clone this branch.
2. Deploy metricbeat in Kubernetes.
3. Check `state_namespace` documents are there.

## Related issues

Relates to https://github.com/elastic/elastic-agent/issues/3100.


## Screenshots

![image](https://github.com/elastic/beats/assets/113898685/797761a7-c8e2-49d4-ba90-00eaaa0b0dcf)
